ikasaneip/component/endpoint/mongo4-endpoint/src/main/java/org/ikasan/component/endpoint/mongo/MongoClientFactory.java

```diff
@@ -0,0 +1,81 @@
+package org.ikasan.component.endpoint.mongo;
```

Comment:
We meantioned this before can we use different package for mongo4

Reply:
done, please have a look